pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 ]
 description = "Ophyd devices and other utils that could be used across DLS beamlines"
 dependencies = [
-    "ophyd@git+https://github.com/DominicOram/ophyd@31a1762435bcb275d2555068233549885eab02e7", # Switch back to just "ophyd" once ophyd#1148, ophyd#1155, ophyd#1140 merged.
```

Comment:
> once ophyd#1148 [...] merged

https://github.com/bluesky/ophyd/pull/1148 isn't merged yet, we still need to point at a branch with that on it I think

Reply:
oops... I thought this had been included in 1.9.0, sorry